### DISCUSSION TITLE
about filter_inquiry_scan_result

### DISCUSSION THREAD

**Discussion by nonzhe** [pschatzmann/ESP32-A2DP#364] [Q&A] [answered]
It's a great project. I successfully sent the bt signal to the bt speaker, but I need to manually specify the name of the bt speaker, which is very inconvenient. Can I search the name of the bt speaker automatically? I see this function “filter_inquiry_scan_result”  in the source program. I don't know how to use it? Thank you.

**Comment by pschatzmann** [comment, marked as answer]
see https://github.com/pschatzmann/ESP32-A2DP/pull/321
There is even an [example](https://github.com/pschatzmann/ESP32-A2DP/blob/main/examples/bt_music_sender_ssid_callback/bt_music_sender_ssid_callback.ino)!

**Reply by nonzhe** [reply]
Thank you very much!
